### PR TITLE
Yel 1810

### PR DESCRIPTION
**Pull request by duncan9562**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -25,7 +25,7 @@ exec > >(tee /var/log/user-data.log|logger -t user-data ) 2>&1
 echo "#Pull values from Parameter Store and save to profile"
 touch /home/tableau_srv/env_vars.sh
 echo "
-export S3_HAPROXY_CONFIG_BUCKET=${var.s3_haproxy_config_bucket}
+export S3_HAPROXY_CONFIG_BUCKET=${var.s3_httpd_config_bucket}
```
Comment (david-wirelab) [inline main.tf]: To be in line with all changes the variable name may also need changing from `S3_HAPROXY_CONFIG_BUCKET` to `S3_HTTPD_CONFIG_BUCKET`

File: main.tf [hunk 2]
```diff
@@ -213,7 +213,7 @@ exec > >(tee /var/log/user-data.log|logger -t user-data ) 2>&1
...
```
Comment (david-wirelab) [inline main.tf]: To be in line with all changes the variable name may also need changing from `S3_HAPROXY_CONFIG_BUCKET` to `S3_HTTPD_CONFIG_BUCKET`

File: main.tf (hunk 1 above)
Comment (duncan9562) [reply main.tf]: Corrected in latest commit.

File: main.tf (hunk 2 above)
Comment (duncan9562) [reply main.tf]: Corrected in latest commit.

**Pull request merged**